app/server/contactUsApi.ts

```diff
@@ -0,0 +1,135 @@
+import { captureMessage } from "@sentry/node";
+import { Request, Response } from "express";
+import fetch from "node-fetch";
+import { contactUsConfig } from "../shared/contactUsConfig";
+import { ContactUsReq } from "../shared/contactUsTypes";
+import { getContactUsAPIHostAndKey } from "./apiGatewayDiscovery";
+import { log } from "./log";
+
+export const contactUsFormHandler = async (req: Request, res: Response) => {
+  const validBody = parseAndValidate(req.body);
+  if (!validBody) {
+    return res.status(400).send();
```

Comment:
Can you give more information about why it wasn't valid or is that deliberate?

Reply:
This was deliberate. If the body isn't valid it means we have a bug in our code so there's nothing the user can do about it.

Reply:
We did however forgot to log this error to Sentry. I will add it now.